scripts/mkdist.js

```diff
@@ -14,43 +14,40 @@ function mkdist (opts, cb) {
 
   var dist = opts.dist
   var ver = dist.version
-  if (ver.match(/^v/)) ver = ver.substring(1)
```

Comment:
Because `master` as a version breaks it, and that's what we do with Orbit. I removed a line later down that had the `v` hardcoded, so it doesn't make a difference.